pkg/cluster/internal/providers/docker/provider.go

```diff
@@ -100,7 +100,7 @@ func (p *Provider) ListNodes(cluster string) ([]nodes.Node, error) {
 		// filter for nodes with the cluster label
 		"--filter", fmt.Sprintf("label=%s=%s", clusterLabelKey, cluster),
 		// format to include the cluster name
-		"--format", `{{.Names}}`,
+		"--format", `{{.ID}}`,
```

Comment:
other things use the cluster names, this is a breaking API change. please look at the node API

Reply:
That comment doesn't seem right to me

Reply:
see below where we use the output to create a `Node` with a string value, this value is used to implement the `Name()` method which other things key off of.